extensions/EXT/EGL_EXT_query_reset_notification_strategy.txt

```diff
@@ -0,0 +1,78 @@
+Name
+
+	EXT_query_reset_notification_strategy
+
+Name Strings
+
+	EGL_EXT_query_reset_notification_strategy
+
+Contributors
+
+	Simon Zeni <simon.zeni@collabora.com>
+
+Contact
+
+	Simon Zeni <simon.zeni@collabora.com>
+
+Status
+
+	Draft
+
+Version
+
+	Version 0.1, 2023-11-01
+
+Number
+
+	EGL Extension #152
+
+Extension Type
+
+	Client
+
+Dependencies
+
+	Requiers EGL 1.4 and EXT_create_context_robustness.
+
+	The extension is written against the wording of the EGL 1.5
+	specification, but may be implemented against 1.4.
+
+Overview
+
+	This extension allows the user to query the reset notification strategy
+	of a given context using the function eglQueryContext.
```

Comment:
Maybe you could add a bit more motivation to the overview for why this is needed/what problem it solves?